File: KUPC_2015/A.py
# encoding:utf-8
import copy
import numpy as np
import random

T = int(input())
S = []
for i in range(T):
    S.append(list(str(input())))

def solve(tape):
    ans = 0
    i = 0
    position = []
    for i in range(len(tape)-4):
        # print(tape[i:i+5])
        if tape[i:i+5]==list("tokyo") or tape[i:i+5]==list("kyoto"):
            position.append([i,i+4])
    position.sort(reverse=True,key=lambda x:x[1])
    now = len(tape)+10
    # print(position)
    for i in range(len(position)):
        if now>position[i][1]:
            ans += 1
            now = position[i][0]

    return ans

for i in range(T):
    print(solve(S[i]))

File: KUPC_2015/A.cpp
//encoding:utf-8
#include <iostream> //c++
#include<time.h> //時間にかんして
#include<stack> //スタックに関して　最新のデータから捨てる
#include<queue> //キューに関して　最古のデータから捨てる
#include<math.h>
#include<vector>
#include<algorithm> //sort とかにいる

#define ll long long
using namespace std;


int solve(char T){
  int ans = 0;
  cout<<T<<endl;
  return ans;
}


int main(){
  int T;
  cin>>T;
  char S[T];
  for(int i=0;i<T;i++){
    cin>>S[i];
  }
  for(int i=0;i<T;i++){
  cout<<solve(S[i])<<endl;
  }
  return 0;
}
